samples/cpp/benchmark_app/main.cpp

```diff
@@ -278,18 +278,26 @@ int main(int argc, char* argv[]) {
 
             // the rest are individual per-device settings (overriding the values set with perf modes)
             auto setThroughputStreams = [&]() {
-                const std::string key = getDeviceTypeFromName(device) + "_THROUGHPUT_STREAMS";
+                std::string key = getDeviceTypeFromName(device) + "_THROUGHPUT_STREAMS";
                 if (device_nstreams.count(device)) {
                     // set to user defined value
                     auto supported_properties = core.get_property(device, ov::supported_properties);
-                    if (std::find(supported_properties.begin(), supported_properties.end(), key) ==
+                    if (std::find(supported_properties.begin(), supported_properties.end(), key) !=
                         supported_properties.end()) {
-                        throw std::logic_error("Device " + device + " doesn't support config key '" + key + "'! " +
+                        device_config[key] = device_nstreams.at(device);
+                    } else if (std::find(supported_properties.begin(),
```

Comment:
Do these changes are applicable for python benchmark_app?

Reply:
Yes the options should be ranamed in python API to.